src/Aspire.ProjectTemplates/templates/aspire-starter/Aspire-StarterApplication.1.Tests/WebTests.cs

```diff
@@ -23,10 +24,16 @@ public async Task GetWebResourceRootReturnsOkStatusCode()
         {
             clientBuilder.AddStandardResilienceHandler();
         });
+#if (TestFramework == "xUnit.net")
+        appHost.Services.AddLogging(logging => logging.AddXunit(output));
```

Comment:
If we don't want to include an `XunitLogger` implementation in the templates, we could instead add a comment pointing folks to https://www.nuget.org/packages?q=logging+xunit

Reply:
XunitContext?

Reply:
I feel like it is a pretty big chunk of code to include. But I think that there are some benefits to having it here just to make things simple for people. I'm surprised this isn't in a package somewhere.

Reply:
There's lots of them! Just none "official".

Reply:
This amount of code in the template exceeds my comfort level.

Reply:
Yeah it's you I was thinking of 😄 
Let's remove it and add a comment pointing to a nuget query instead.